### ISSUE TITLE
why are you don't fix old friv?

### ISSUE THREAD

**Issue by deividbinev12** [Tsyuay/FrivReborn#7]
you don't know something how to make flash cs6 old friv
you don't work

**Comment by Tsyuay** [comment]
hello there, I know how to use FL CS6 with friv since I made some changes in the SWF. but I dont use that anymore because I just use it for bug fixes. not games. and if you want the Late Night Games back I can modify the SWF a little bit and add it. There's 
a 0.1% chance but I might try it soon. and I know the page was in a hiatus it was because I was working in GDevelop the Panda Fake Project's DCSHG stuff (Yes, I'm who create these fan games) and because of that it was in a hiatus for a long time. but I might back to work at it some days later.

**Comment by deividbinev12** [comment]
Yes i know friv i remember we played i'm too little year old 5 with friv games 10 year ago (2012) the old good day but my mom told me that's flash games 10 year ago

New Friv Website is good okay i miss old friv you don't know how to make late night games if you make extra games? i'm not sure i want fix old friv there is no longer game under old friv why?
